src/types.ts

```diff
@@ -151,6 +151,9 @@ export interface TranslationMetadata {
   };
 }
 
+export type TranslationDict = typeof import("./translations/en.json") &
+  typeof import("../translations/backend/en.json");
```

Comment:
this file is not up to date, and won't be kept up to date, I think we should exclude all keys that come from the backend in `LocalizeKeyExceptions` and remove this file for the type.

Reply:
Exception is already in place. I'll removed them from the dictionary.